### PR TITLE
[refactor] ☕ Short and simple method names

### PR DESCRIPTION
**Pull request by TangoMan75**
[refactor] ☕ Short and simple method names
-------------------------------------------

## 🍬 1. Renamed `findTheLatest()` to `findLast()`

🔥 Affected files:
- 📝 README (line 72)
- 📂 src/AirtableClient (line 80)
- 📂 src/AirtableClientInterface (line 48)
- ✅ tests/AirtableClientTest (lines 211-248)

## 🍩 2. Renamed `addOneRecord()` to `add()`

🔥 Affected files:
- 📝 README (line 74)
- 📂 src/AirtableClient (line 113)
- 📂 src/AirtableClientInterface (line 57)
- ✅ tests/AirtableClientTest (lines 312-363)

## 🍰 3. Renamed `findOneById()` to `find()`

🔥 Affected files:
- 📝 README (line 65)
- 📂 src/AirtableClient (line 65)
- 📂 src/AirtableClientInterface (line 39)
- ✅ tests/AirtableClientTest (lines 267-293)

### CODE REVIEW

File: src/AirtableClient.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function findTheLatest(string $table, $field, ?string $dataClass = null):
     /**
      * {@inheritdoc}
      */
-    public function addOneRecord(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
+    public function add(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
```
Comment (liorchamla) [inline src/AirtableClient.php:100]: Wait wait, I know this is not the point of this PR but .. This method (add) should take $fields as an array OR an object which would be denormalized as an array no ?

**Comment by liorchamla** [comment]
Great job of simplification here :)

I just noticed (and you will see it in the comments) that the add method of AirtableClient (and so AirtableClientInterface) should not force us to send an array for  $fields, but an array OR an object.

Well played for the rest :)

File: src/AirtableClient.php (hunk 1 above)
Comment (TangoMan75) [reply src/AirtableClient.php:100]: True... So, what should be the appropriate type hint ? I'm not sure PHP handles that kind of union types... Not before version 8 anyway... Do you want me to change this ? Or maybe create another issue ?

Comment (liorchamla) [reply src/AirtableClient.php:100]: Since this is not the point of this PR, maybe we should wait for the merge THEN address this problem :x

Comment (yoanbernabeu) [reply src/AirtableClient.php:100]: I suggest that we look at this in another PR?

**Review by yoanbernabeu — COMMENTED** [review]
Thanks for this PR!
It is much simpler and more understandable.

I'll merge first [#16](https://github.com/yoanbernabeu/Airtable-Client-Bundle/pull/16), we'll see if there are any conflicts to deal with then

It's OK for you ?

**Comment by yoanbernabeu** [comment]
I'm back after a very busy work week!

This proposal is really good and it will simplify the use of the Bundle.
Do you have time to resolve conflicts since the merge of PR #16 and #21 ?

Thank you !

**Comment by TangoMan75** [comment]
Sure, I'll work on it... You're welcome

**Comment by TangoMan75** [comment]
Conflicts solved... Ready for merge

**Comment by TangoMan75** [comment]
Hi @yoanbernabeu 👋
I really enjoyed your last video about nocodb 👍 Great tool, thank you !
Are you still interested in my PR ? There are no more conflicts with main branch

**Review by yoanbernabeu — APPROVED** [review]
Thanks for the changes.
I run after time ;-)
I hope I have time this weekend to merge definitely

**Pull request merged**